internal/pkg/otel/translate/otelconfig.go

```diff
@@ -299,6 +299,7 @@ func getReceiversConfigForComponent(
 	receiverConfig := map[string]any{
 		// just like we do for beats processes, each receiver needs its own data path
 		"path": map[string]any{
+			"home": paths.Components(),
```

Comment:
This isn't going to fix it if somebody uses `metricbeatreceiver` in a standalone config, though I think that problem will fix itself if the `elastic-otel-collector` binary goes into the components directory where beats is today so maybe we don't need to do anything to correct that later.

https://www.elastic.co/docs/reference/beats/filebeat/configuration-path#_home

> The home path for the Filebeat installation. This is the default base path for all other path settings and for miscellaneous files that come with the distribution (for example, the sample dashboards). If not set by a CLI flag or in the configuration file, the default for the home path is the location of the Filebeat binary.

I think this problem exists because the beat executes in the elastic-agent binary right.

Reply:
> This isn't going to fix it if somebody uses `metricbeatreceiver` in a standalone config, though I think that problem will fix itself if the `elastic-otel-collector` binary goes into the components directory where beats is today so maybe we don't need to do anything to correct that later.

Yes, moving `elastic-otel-collecotr` to components dir would also "fix" this and would make it behave the same as agentbeat.

Reply:
> If the module subdirectory is in the same directory as the executable you don't have to specify path.home, if you move it then you do. So I don't think this will require any config translation

The difference is that we moved it, not the user, so we force everyone to define path.home. Let's fork fixing standalone metricbeat receiver into a separateissue.